Makefile

```diff
@@ -301,9 +301,13 @@ build-image:
 generate-k8s:
 	$(Q)GOCACHE=$(GOCACHE) operator-sdk generate k8s
 
+build-openapi-gen-cli: $(OUTPUT_DIR)/openapi-gen
+	go build -o $(OUTPUT_DIR)/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen
```

Comment:
should we not use `GOCACHE=$(GOCACHE)`  as everywhere else?

Reply:
Done. a7c02c08bc168b9d04db6b46d9510794867e0041